frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -15,12 +15,9 @@ import {
   makeReduxID,
   units,
 } from '@console/internal/components/utils';
+import { NetworkAttachmentDefinitionModel } from '@console/network-attachment-definition-plugin/src/models';
```

Comment:
Wouldn't it make sense to move the model to `console-shared` ?

if not, the `@console/network-attachment-definition-plugin` should be listed in `kubevirt-plugin/package.json`.

Reply:
I had this conversation with someone else (can't remember who at the moment). It was suggested I move it to the `network-attachment-definition-plugin`, but I agree that it makes more sense to move shared content to `console-shared`, so I did that. =)